conda/environment.yml

```
@@ -2,12 +2,11 @@
 # Then run `./lock-environment.sh` to generate reproducible lock files
 
 # !!! If you are developing the website locally use the existing lock files
-# for your platform (`conda create --name hackweek --file conda-osx-64.lock`)
-name: hackweek
+# for your platform (`conda create --name cryocloud --file conda-linux-64.lock`)
+name: cryocloud
 channels:
   - conda-forge
-dependencies:
-  - cookiecutter
+dependencies: 
```

Extra whitespace removal
```suggestion
dependencies:
```